calendar/calendar-util/util/src/java/org/sakaiproject/util/CalendarUtil.java

```diff
@@ -43,6 +37,8 @@
 import org.joda.time.format.DateTimeFormatter;
 import org.joda.time.format.DateTimeFormatterBuilder;
 
+import static org.sakaiproject.util.CalendarEventType.getLocalizedLegendFromEventType;
```

Comment:
Why the static import for just this one method but not the others you use? You could just switch to a normal import.